### PR TITLE
Covered 2 items for Times issue 147

### PR DESCRIPTION
**Pull request by ijlee2**
- Alberto's blog post on autosave in Ember Octane
- Embroider and memoization RFCs entering their final comment period

### CODE REVIEW

File: source/2020-05-08-the-ember-times-issue-147.md [hunk 1]
```diff
@@ -37,25 +39,17 @@ Check out the full list of features and improvements over at the project's [chan
...
-## [Section title in sentence case 🐹](section-url)
+## [Embroider & memoization RFCs entered final comment period 💬](https://github.com/emberjs/rfcs/pull/507)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
-
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+In case you missed, the RFCs for [Embroider](https://github.com/emberjs/rfcs/pull/507) and [memoizing tracked properties](https://github.com/emberjs/rfcs/pull/615) entered their final comment period (FCP) on May 1.
```
Comment (ijlee2) [inline source/2020-05-08-the-ember-times-issue-147.md:52]: I don't know if an FCP can last for more than 7 days. I assumed the due dates for these RFCs are today and wasn't sure what more I could talk about. 😓

Comment (jayjayjpg) [reply source/2020-05-08-the-ember-times-issue-147.md:52]: Yeah, I agree, I also assume that the comment period will close very soon, so I think this wording works great!

File: source/2020-05-08-the-ember-times-issue-147.md [hunk 2]
```diff
@@ -37,25 +39,17 @@ Check out the full list of features and improvements over at the project's [chan
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Auto save en Ember Octane 💾](https://betocantu93.com/autosave-in-ember-octane/)
```
Comment (jayjayjpg) [inline source/2020-05-08-the-ember-times-issue-147.md:42]: Nice touch with the :es: title :sparkles:

**Review by jayjayjpg — APPROVED** [review]
This reads great! Thank you for another :sparkles: writeup for the Ember Times this week :rainbow:

**Pull request merged**